content/en/docs/concepts/workloads/pods/pod-quality.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Pod Quality of Service Classes
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+This page explains how Kubernetes assigns Quality of Service (QoS) classes to
+Pods as a side effect of specifying resource constraints on Containers. QoS
+classes are by Kubernetes to make decisions about which Pods to evict when
+there are not enough resources on a Node.
+
+<!-- body -->
+
+## Quality of Service Classes
+
+Because Kubernetes co-locates all Containers within a given {{< glossary_tooltip text="Pod" term_id="pod" >}}
+to the same {{< glossary_tooltip text="Node" term_id="node" >}}, it uses a Pod-level
+classification to determine how to meet
+[resource requirements](docs/concepts/configuration/manage-resources-containers/).
+This is known as {{< glossary_tooltip text="Quality of Service" term_id="qos-class" >}}
+(QoS) class. Kubernetes assigns every Pod a QoS class based on the resource requests
+and limits of its component {{< glossary_tooltip text="Containers" term_id="container" >}}.
+QoS classes are used by Kubernetes to decide which Pods to evict from a Node experiencing
+[Node Pressure](/docs/concepts/scheduling-eviction/node-pressure-eviction). The possible
+QoS classes are Guaranteed, Burstable, and BestEffort. When a Node runs out of resources,
+Kubernetes will first evict BestEffort Pods running on that Node followed by Burstable and
+Guaranteed Pods. When this eviction is due to resource pressure, only Pods exceeding their
+resource requests are candidates for eviction.
+
```

Comment:
Which is right?
```suggestion
The kube-scheduler also considers QoS class when selecting which Pods to 
[preempt](/docs/concepts/scheduling-eviction/pod-priority-preemption/#preemption) in case
the cluster does not have enough resources to run all the defined Pods.
```
?
```suggestion
However, the kube-scheduler does not consider QoS class when selecting which Pods to 
[preempt](/docs/concepts/scheduling-eviction/pod-priority-preemption/#preemption).
Preemption can occur when a cluster does not have enough resources to run all the Pods
you defined.
```

Reply:
Second one, right?